### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]:Jaws and NVDA are not announcing the name of the edit in "Push Image" dialog

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1467]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Create a new java project.
3. Tab to and select Project ->Azure->Push Image.
4. In "Push Image" dialog ,  tab to Docker File , Server URL ,Username ,Image and Tag edit boxes and observe the behaviour of Jaws and NVDA

**Actual Result:**
Jaws and NVDA is not announcing the name of the edit boxes

**Jaws behaviour:**
Edit
Type in text
Edit
Type in text
Edit
Type in text
Edit
Type in text

**NVDA Behaviour:** 
edit
blank
edit
blank
edit
blank

**Expected Result:**
Jaws and NVDA should announce the name of the edit fields
Name should be 
Docker File
Server URL
Username
Image and Tag

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![push image-edit-name-empty](https://user-images.githubusercontent.com/22956130/36102476-70961a08-1032-11e8-9af1-5c3086d0aab4.jpg)
![251277a5](https://user-images.githubusercontent.com/22956130/36102489-741eceea-1032-11e8-891e-5292c783b9ea.jpg)
![push image-edit-explorer](https://user-images.githubusercontent.com/22956130/36102490-76e5aea0-1032-11e8-9ea7-a8c2c30d81fe.jpg)

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749925
